index.bs

```diff
@@ -952,10 +948,10 @@ and runs the associated step:
 :: Return {{DONE}}
 
 The <dfn attribute for=FileReader>result</dfn> attribute's getter,
-when invoked, must return the [=context object=]'s [=FileReader/result=].
+when invoked, must return [=this=]'s [=FileReader/result=].
 
 The <dfn attribute for=FileReader>error</dfn> attribute's getter,
-when invoked, must return the [=context object=]'s [=FileReader/error=].
+when invoked, must return [=this=]'s [=FileReader/error=].
 
 <div algorithm="read operation">
 A {{FileReader}} |fr| has an associated <dfn id=readOperation>read operation</dfn> algorithm,
```

Comment:
WDYT about replacing |fr| references in this algorithm with [=this=] ?

Reply:
And same for "A {{Blob}} |blob| ..." above (line 266, which I can't comment on here)

Reply:
[=this=] only has meaning inside method steps/getter steps/setter steps/constructor steps, so I don't think that works.